hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -3027,7 +3047,7 @@ func (r *HostedClusterReconciler) delete(ctx context.Context, hc *hyperv1.Hosted
 		return false, err
 	}
 
-	exists, err := deleteAWSEndpointServices(ctx, r.Client, controlPlaneNamespace)
+	exists, err := deleteAWSEndpointServices(ctx, r.Client, hc, controlPlaneNamespace)
```

Comment:
Even with this change, we would still get an exists == true and keep waiting, no?
All it looks like we're doing is skipping the delete call

Reply:
Please ignore me